s2core/src/test/scala/org/apache/s2graph/core/CommonTest.scala

```diff
@@ -0,0 +1,13 @@
+package org.apache.s2graph.core
```

Comment:
we missed out Apacche license header here. please ```sbt runRatTask``` and examine rat.out to check apache license header.